src/hipscat/pixel_math/cone_filter.py

```diff
@@ -27,13 +26,15 @@ def filter_pixels_by_cone(pixel_tree: PixelTree, ra: float, dec: float, radius:
         [PixelAlignment.PRIMARY_ORDER_COLUMN_NAME, PixelAlignment.PRIMARY_PIXEL_COLUMN_NAME]
     ]
     filtered_pixels_df = pixels_df.drop_duplicates()
-    partition_info_df = filtered_pixels_df.rename(
-        columns={
-            PixelAlignment.PRIMARY_ORDER_COLUMN_NAME: PartitionInfo.METADATA_ORDER_COLUMN_NAME,
-            PixelAlignment.PRIMARY_PIXEL_COLUMN_NAME: PartitionInfo.METADATA_PIXEL_COLUMN_NAME,
-        }
-    )
-    return partition_info_df.reset_index(drop=True)
+
+    pixel_list = [
+        HealpixPixel(order, pixel)
+        for order, pixel in zip(
+            filtered_pixels_df[PixelAlignment.PRIMARY_ORDER_COLUMN_NAME],
+            filtered_pixels_df[PixelAlignment.PRIMARY_PIXEL_COLUMN_NAME],
+        )
+    ]
+    return pixel_list
```

Comment:
We need to update this method's signature to return `List[HealpixPixel]` and correct the docstring description because it still mentions we return a dataframe. Other than that, everything looks great!